client/my-sites/people/people-list-item/test/index.jsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import { shallow } from 'enzyme';
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import useFakeDom from 'test/helpers/use-fake-dom';
+import useMockery from 'test/helpers/use-mockery';
+
+describe( 'PeopleListItem', function() {
+	let configMock,
+		PeopleListItem,
+		CompactCard;
+
+	useFakeDom();
+
+	useMockery( mockery => {
+		configMock = sinon.stub();
+		configMock.isEnabled = sinon.stub();
+		mockery.registerMock( 'config', configMock );
+		mockery.registerMock( 'lib/analytics', {} );
+	} );
```

Comment:
With the Redux analytics actions and with our own little browser mock in the component, we can probably eliminate these mocks entirely, leaving much simpler tests.

in fact, if we import the actual named export for the component instead of the `connect()`ed version (including the change I made above to the `navigateToUser` function) then we have zero problems and can do a proper import

```js
import { PeopleListItem } from '..';
```